pages/topics/index.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import Link from 'next/link';
+import { RightArrowIcon } from '../../icons/index';
+
+const Topics = ({ topics }) => (
+  <section className="w-full py-2 text-gray-700">
+    <h1 className="mt-10 text-3xl sm:text-4xl font-medium text-gray-900 text-center">Topics</h1>
+    <div className="mt-10">
+      {topics ? (
```

Comment:
What if we make sure that `topics` is a list. That way, we can just map and don't ask if it's truthy.

We can do this if we lift the condition, like so:
```suggestion
    props: {
      topics: topics || [],
    },
```

Reply:
Also, why do you need the ternary (in case we keep it), wouldn't it be just fine if you short-circuit?

```suggestion
      {topics && (...
```

And then just remove the `<span />`, always try to keep the code terse.